integration/airflow/tests/integration/docker-compose.yml

```diff
@@ -1,4 +1,36 @@
 version: "3.7"
+
+x-airflow-common: &airflow-common
```

Comment:
Should this be `airflow-common` instead of `x-airflow-common`?

Reply:
Minor: I'd use `airflow-base` as everything else _extends_ / adds on top of the config ;)

Reply:
`x-` is because

```
ERROR: The Compose file './docker-compose.yml' is invalid because:
Invalid top-level property "airflow-base". Valid top-level sections for this Compose file are: version, services, networks, volumes, secrets, configs, and extensions starting with "x-".
```

So, moved to `x-airflow-base`.